src/features/locators/utils/runCssSelectorGeneration.ts

```diff
@@ -0,0 +1,7 @@
+import { sendMessage } from "../../../pageServices/connector";
+import { Locator } from "../types/locator.types";
+
+export const runCssSelectorGeneration = async (locators: Locator[]) => {
+  sendMessage.generateSelectorGroupByHash({ elements: locators, fireCallbackMessage: true });
+  return Promise.resolve("started");
```

Comment:
I'd suggest use `LocatorsGenerationStatus` enum here like in `src/features/locators/utils/runXpathGeneration.ts`

Reply:
sure. done